### ISSUE TITLE
More settings / user defined defaults

### ISSUE THREAD

**Issue by Marvin182** [tensorflow/tensorboard#294]
I would love to see have a way configure how my TensorBoards look like when I open them. While experimenting it's not uncommon to open/close many TensorBoards a day and the defaults (ignore outliers,  smoothing=0.6, filter=.*) don't fit anyone. I can think of two solutions:
1) Remember the last choice of the user.
2) Extend the settings menu to allow overriding the defaults (and reset to them).

**Comment by wchargin** [comment]
Agreed: I'd like to do this. Let's think about the best ways to do so.

You mention that you open/close many TensorBoards per day. In your experience, will you usually want the same settings across all of them? That is, do you expect option (1) to work well in practice?

I don't follow what you mean by option (2). Are you suggesting that we move the individual dashboards' display options into the settings dialog, which currently only has the auto-reload toggle? How would this alone help? (Also, on the subject: currently each dashboard "owns" everything below the big orange bar, but TensorBoard itself owns the settings pane. It'd be possible to change this, but I'd prefer not to.)

**Comment by danijar** [comment]
I'm creating multiple TensorBoard instances every day and every time have to make the following changes via the UI:

- Set smoothing value
- Change tooltip sorting to `nearest`
- Disable ignoring outliers in chart scaling
- Disable automatic data reload
- Change the pagination limit to fit my screen

It would be very useful if those could be set via the command line, so I can create an alias and don't have to worry about it every time. Since the TensorBoards are running on different servers, remembering my last choice would not work here.

**Comment by DM1122** [comment]
Completely agree with the need for this. Has any progress been made on overriding Tensorboard's default config?
